commands/access/add.js

```diff
@@ -36,7 +42,7 @@ function* run(context, heroku) {
       },
       body: {
         user: context.args.email,
-        privileges: privileges.split(",")
+        privileges: privileges
```

Comment:
This is for if you don't take Naaman's feedback:
If you are pushing "view" after you've comma split privileges (which is a string), then there won't be a comma between the prior privileges and view.